WordPress/src/test/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModelTest.kt

```diff
@@ -18,22 +20,25 @@ class WPMainActivityViewModelTest {
 
     private lateinit var viewModel: WPMainActivityViewModel
 
+    @Mock
+    private lateinit var appPrefsWrapper: AppPrefsWrapper
+
     @Before
     fun setUp() {
-        viewModel = WPMainActivityViewModel()
+        viewModel = WPMainActivityViewModel(appPrefsWrapper)
         viewModel.start(true)
     }
 
     @Test
     fun `fab visible when asked`() {
         viewModel.onPageChanged(true)
-        assertThat(viewModel.showMainActionFab.value).isEqualTo(true)
+        assertThat(viewModel.showMainActionFab.value?.isFabVisible).isEqualTo(true)
     }
 
     @Test
     fun `fab hidden when asked`() {
         viewModel.onPageChanged(false)
-        assertThat(viewModel.showMainActionFab.value).isEqualTo(false)
+        assertThat(viewModel.showMainActionFab.value?.isFabVisible).isEqualTo(false)
```

Comment:
You can add 2 more tests here for the tooltip field :-)

Reply:
Correct 😊....see here 5511a9a541dc4c8e8bafcad7bc8ddc3a40f996a1